homeassistant/components/recorder/models.py

```diff
@@ -666,3 +666,19 @@ def __eq__(self, other: Any) -> bool:
             and self.state == other.state
             and self.attributes == other.attributes
         )
+
+
+class LazyMinimalState(LazyState):
+    """A lazy minimal version of core State."""
+
```

Comment:
The object creation overhead might be a bit much here. Probably need to override it a bit more to only do a minimal init